webui/react/src/services/decoder.ts

```diff
@@ -20,7 +20,7 @@ import {
 } from 'types';
 import { capitalize } from 'utils/string';
 
-import { Determinedexperimentv1State, V1Experiment } from './api-ts-sdk';
+import * as AB from './api-ts-sdk'; // API Bindings
```

Comment:
I saw references of this as `Api`. I personally like `ApiSdk`.

Reply:
about the  generated code. I think I named it poorly. people have mentioned that it's more of a binding than a SDK

Reply:
considering that, we probably don't want sdk in the name

Reply:
`ApiBindings` would be kind of long for something we plan to repeated `n`+ (https://github.com/determined-ai/determined/pull/1459#discussion_r508069194)  times